charts/kong/README.md

```diff
@@ -288,6 +293,124 @@ scenario and several settings that are useful when using multiple controllers:
 Standalone controllers require a database-backed Kong instance, as DB-less mode
 requires that a single controller generate a complete Kong configuration.
 
+### Hybrid mode
+
+Kong supports [hybrid mode
+deployments](https://docs.konghq.com/2.0.x/hybrid-mode/) as of Kong 2.0.0 and
+Kong Enterprise 2.1.0. These deployments split Kong nodes into control plane
+(CP) nodes, which provide the admin API and interact with the database, and
+data plane (DP) nodes, which provide the proxy and receive configuration from
+control plane nodes.
+
+The chart does not support deploying control and data plane nodes in a single
+release. Users should [create separate releases for each node
+type](#separate-admin-and-proxy-nodes), i.e. use separate control and data
+plane values.yamls that are installed separately.
```

Comment:
Suggested reword (to focus this section on "what hybrid mode is" and "how to achieve hybrid mode":

> [Hybrid mode](https://docs.konghq.com/2.0.x/hybrid-mode/) is a deployment pattern supported by Kong as of Kong 2.0.0 and Kong Enterprise 2.1.0. Deploying Kong in hybrid mode means splitting Kong nodes into **control plane (CP) nodes** (which provide the admin API and interact with the database), and **data plane (DP) nodes** (which provide the proxy and receive configuration from control plane nodes).
> 
> You can deploy Kong in hybrid mode by [creating separate Helm releases for each node type](#separate-kong-functions-across-nodes): one Helm release for control plane nodes, and one for data plane nodes.

Reply:
accidentally marked as resolved?

Reply:
That section changed a fair bit for other reasons, so if there are still changes needed, it should be in a new comment.